specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/Databases.json

```diff
@@ -708,61 +699,6 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/import": {
```

Comment:
This is just moving to servers.json. no any change, right?

Reply:
correct